test/test_config_base.py

```diff
@@ -27,11 +27,13 @@
 from apprise.AppriseAsset import AppriseAsset
 from apprise.config.ConfigBase import ConfigBase
 from apprise import ConfigFormat
-from apprise import plugins
 import yaml
 
 # Disable logging for a cleaner testing output
 import logging
+
+from apprise.plugins.NotifyEmail import NotifyEmail
```

Comment:
To back my earlier comment, here is a perfect example where i do hardcode tests against some plugins (contradicting myself), however as stated, Email sits in one of those categories that isn't going anywhere . (ignore this review comment, it's already resolved, but i just wanted to clarify :slightly_smiling_face: )

Reply:
Thanks!